manifest.json

```diff
@@ -4,7 +4,7 @@
 	"name": "bugzilla-github-extension",
 	"short_name": "BZ<->GH",
 	"description": "This extension integrates GitHub and Bugzilla",
-	"version": "2.0.1",
+	"version": "2.0.1.1",
```

Comment:
This should be 2.0.2. We try to follow this: https://semver.org/, so this would be a minor bug fix patch.